src/components/LanguageSelector.vue

```diff
@@ -41,6 +47,18 @@ const languageDropdownOptions = computed(() => {
     };
   });
 });
+
+async function onLanguageChange(event) {
+  sessionStorage.setItem(`${isLevante ? 'levante' : 'roar'}PlatformLocale`, event.value);
+
+  console.log('event', event.value);
+
```

Comment:
Could you please clarify why this change listener is needed and why we need to write to session storage?

Reply:
Max and I discussed this on a call. The handler for this event has been moved up to the container. This also fixes an Issue with the NavBar component test.